src/libLookup/Lookup.cpp

```diff
@@ -1670,17 +1671,32 @@ bool Lookup::ProcessGetCosigsRewardsFromSeed(
   const auto& microblockInfos = txblkPtr->GetMicroBlockInfos();
   std::vector<MicroBlock> microblocks;
   for (const auto& mbInfo : microblockInfos) {
+    if (mbInfo.m_shardId ==
+        m_mediator.m_ds->GetNumShards()) {  // ignore ds microblock
+      continue;
+    }
     MicroBlockSharedPtr mbptr;
-    if (!BlockStorage::GetBlockStorage().GetMicroBlock(mbInfo.m_microBlockHash,
-                                                       mbptr)) {
-      cout << "Could not get MicroBlock " << mbInfo.m_microBlockHash << endl;
+    retryCount = 5;
```

Comment:
For the hard-code retryCount, maybe we can make it meaningful, and also combine with previous retryCount (in line 1655) as well.

Reply:
retryCount has to be used separately for txBlk and microblocks. For meaninfulness, you meant move it to local constant?

Reply:
In my opinion, no need to put into constant file. Just give a meaningful name by using `#define` / `const variable` should be enough